ManagedClientConsoleAppSample/Program.cs

```diff
@@ -11,10 +12,11 @@ class Program
         //============= Config [Edit these with your settings] =====================
         internal const string vstsCollectionUrl = "https://myaccount.visualstudio.com"; //change to the URL of your VSTS account; NOTE: This must use HTTPS
         // internal const string vstsCollectioUrl = "http://myserver:8080/tfs/DefaultCollection" alternate URL for a TFS collection
+        internal const string clientId = "0fa17cf4-f75c-4185-ab9a-7c5ea47ca073"; //change to your app registration's Application ID
```

Comment:
Do we even want to broadcast that approach? Shouldn't we be pushing users to use ADAL the recommended way instead of learning a 1-off approach that only works for VSTS?